Taarafo.Core/Models/Events/Event.cs

```diff
@@ -18,6 +18,6 @@ public class Event
         public Guid CreatedBy { get; set; }
         public DateTime CreatedDate { get; set; }
         public Guid UpdatedBy { get; set; }
-        public DateTime MyProperty { get; set; }
+        public DateTime? UpdatedDate { get; set; }
```

Comment:
Please remove the `?` as this field is mandatory.  At time of creation, it will have the same value as `CreatedDate`.